### PR TITLE
Fully remove old l10n uses and machinery

### PR DESCRIPTION
**Pull request by robhudson**
## Description

This work is slowly updating templates to remove uses of the old l10n machinery.

- [x] Remove all uses of `_(...)` and `_lazy(...)`
- [x] Remove all uses of `{% trans %}` tags
- [x] Remove old l10n machinery

## Issue / Bugzilla link

#9810

### CODE REVIEW

File: bedrock/firefox/templates/firefox/testflight.html [hunk 1]
```diff
@@ -27,8 +27,8 @@
 {% block content %}
 <main role="main">
   {% call hero(
-  title=_('Help shape the future <br>of Firefox for iOS'),
-  desc=_('Sign up to test pre-release versions to make Firefox even better.'),
+  title='Help shape the future <br>of Firefox for iOS',
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/testflight.html:30]: image

**Review by alexgibson — CHANGES_REQUESTED** [review]
This is a great legacy of tech debt payoff, nice job! 💯 

This is a pretty huge set of changes, but I took a pass through the most high visibility pages and spotted a few errors / fixups, but mostly everything is looking a-ok 👍 

A couple of re-occuring gotchas to look out for:

- Strings defined as veriables using `set` might need escaping if the include HTML markup.
- Same for strings passed as variables to Jinja macros / helpers.

File: bedrock/settings/base.py [hunk 2]
```diff
@@ -99,7 +99,7 @@ def data_path(*args):
 
 # If you set this to False, Django will make some optimizations so as not
 # to load the internationalization machinery.
-USE_I18N = True
+USE_I18N = False
```
[the remark this answers is not in the document]
Comment (robhudson) [reply bedrock/settings/base.py:102]: Yes, the Django i18n machinery is the l10n code that this removes. Django doesn't have integration with Fluent so we can safely disable this.

**Comment by robhudson** [comment]
I found all the cases where `{% set ...` had HTML tags. I will do another pass looking for calls to macros in the morning. I appreciate the catch on those.

**Review by alexgibson — APPROVED** [review]
I took another pass and all of the updates look good. I didn't spot any other occurences of formatting / escaping errors. If there are some, they are likely in rogue / minor pages and can be fixed easily enough in a followup.

Would begood to make sure we get a passing integration test run before merging just to be extra safe.

Nice work!

**Review by pmac — APPROVED** [review]
This was a lot bigger than I imagined it would be. Very nice work! Thanks 🥇

**Pull request merged**